dev/benchmarks/complex_layout/windows/flutter/CMakeLists.txt

```diff
@@ -96,7 +96,7 @@ add_custom_command(
   COMMAND ${CMAKE_COMMAND} -E env
     ${FLUTTER_TOOL_ENVIRONMENT}
     "${FLUTTER_ROOT}/packages/flutter_tools/bin/tool_backend.bat"
-      windows-x64 $<CONFIG>
+      ${FLUTTER_TARGET_PLATFORM} $<CONFIG>
```

Comment:
We will want to create a migration to automate this our users. Here's the corresponding Linux migration: https://github.com/flutter/flutter/blob/b2e96cd75b91ef2599b3eb36c4c95da141b6ee87/packages/flutter_tools/lib/src/migrations/cmake_custom_command_migration.dart#L56-L65

Reply:
Sounds good, I didn't see there was this tool. The related change in gallery example application change mention they use flutter create instead, and updated files from there.